code/game/gamemodes/changeling/powers/fakedeath.dm

```diff
@@ -18,7 +18,7 @@
 	user.update_stat()
 	user.update_canmove()
 
-	addtimer(src, "ready_to_regenerate", LING_FAKEDEATH_TIME, TIMER_NORMAL, user)
+	addtimer(src, "ready_to_regenerate", LING_FAKEDEATH_TIME, TIMER_UNIQUE, user)

```

Comment:
Players where previously able to keep the `alert()` open to call the `addtimer` again, which previously caused it to simply add two timers, allowing changelings to revive themselves whilst alive.